py-polars/requirements-dev.txt

```diff
@@ -19,10 +19,14 @@ adbc_driver_sqlite; python_version >= '3.9' and platform_system != 'Windows'
 connectorx==0.3.2a7  # Latest full release is broken - unpin when 0.3.2 released
 cloudpickle
 fsspec
+s3fs[boto3]
 
 # Tooling
+flask!=2.2.0,!=2.2.1  # Required for moto.server w/o installing all moto[server] dependencies
+flask-cors  # Required for moto.server w/o installing all moto[server] dependencies
 hypothesis==6.82.0
 maturin==1.1.0
+moto[s3]==4.1.13  # Need moto.server to mock s3fs - see aio-libs/aiobotocore#755
```

Comment:
As commented here, we need `moto.server`, not only `moto[s3]`. But `moto[server]` is a total dependency mess due to cloudwatch support - especially it pins `pydantic` to `~1.8`, breaking other tests. We only need S3 part of the server, a brainless pass of manually install `flask` is taken for now.

Reply:
Could we maybe elide this problem with a moto github action? https://github.com/getmoto/moto

Reply:
I've thought a bit and feels like I don't follow what you mean, do you mean launching moto server outside test venv (w/ moto github action) and removing moto dependencies from `requirements-dev.txt`? Doesn't it make `test_cloud.py` not running under local dev setup?